src/sys/ioctl/linux.rs

```diff
@@ -41,6 +41,7 @@ mod consts {
     target_arch = "s390x",
     target_arch = "x86_64",
     target_arch = "aarch64",
+    target_arch = "loongarch64",
```

Comment:
Could you guys please check what are currently supported and what are not? And, if there is still a lot of work to do, and you guys don't mind, a tracking issue would be appreciated:)